src/cloudai/_core/installables/hf_model.py

```
@@ -0,0 +1,135 @@
+
+        return InstallStatusResult(True)
+
+    def uninstall(self, installer: "BaseInstaller") -> InstallStatusResult:
+        if not self._is_hf_home_accessible(installer):
+            return self._assume_available_on_execution_nodes(installer, "removal")
+
+        hf_home_path = installer.system.hf_home_path
+        logging.debug(f"Removing HF model {self.model_name} from {hf_home_path / 'hub'}")
+        res = self.is_installed(installer)
+        if not res.success:
+            return InstallStatusResult(True, f"HF model {self.model_name} is not downloaded.")
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)